src/ubuntu/manifest.json

```diff
@@ -15,6 +15,9 @@
                 "ubuntu-20.04-coredeps-local": {
                   "isLocal": true
                 }
+              },
+              "buildArgs": {
+                "POWERSHELL_TAG": "lts-7.2-ubuntu-20.04"
```

Comment:
Would it make sense to define a versions manifest to start putting these types of valuesin?  The benefit is to ease maintenance by getting them next to each other.  Possibly the value could be decomposed so that the pwsh version is a shared value.

Reply:
Actually, I'm kind of inclined to just change it to a non-version floating tag: `lts-ubuntu-20.04`. That's essentially the same as installing it via package manager as far what version you end up getting: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/cbf67da80a106ac8c3114a75d07bf516eebae9a5/src/ubuntu/22.04/debpkg/amd64/Dockerfile#L23

Reply:
I've updated the tag usage to get the latest PowerShell version available. This aligns with the version that gets installed by other Dockerfiles that are installing from PMC. This will give you PowerShell 7.4 in both cases.

There is some cleanup of other PowerShell usage amongst the Dockerfiles but that's outside the scope of coredeps so I'll address those separately.